src/PrestaShopBundle/Install/Database.php

```diff
@@ -104,6 +104,15 @@ public function testDatabaseSettings($server, $database, $login, $password, $pre
             }
         }
 
+        if (!$errors) {
+            if (($select_error = Db::checkSelectPrivilege($server, $login, $password, $database, $prefix)) !== true) {
```

Comment:
Is it possible to merge those two consecutive conditions into a single one?

Reply:
:heavy_check_mark: